### PR TITLE
Create documentation to authenticate with LDAP

### PR DESCRIPTION
**Pull request by RyanHir**
### Motivation and context
Previously, it was quite difficult to find up-to-date information on how to authenticate users using LDAP. This PR is attempting to centralize knowledge that is spread over different issues in the issue tracker of the old repo. I am aiming to close https://github.com/openvinotoolkit/cvat/issues/311 with this.

### How has this been tested?
Involving a Windows Server VM and a AlmaLinux 8 VM, I was able to test Active Directory and FreeIPA.

#### Tasks to complete before merge
- [x] Write Active Directory (AD) config
- [x] Write FreeIPA config
- [x] Write instructions to modify `docker-compose.override.yml`
- [x] Validate AD config for auth
- [x] Validate FreeIPA config for auth
- [x] Validate AD config for group assignment
  - Needs https://github.com/cvat-ai/cvat/issues/40 to be addressed
- [x] Validate FreeIPA config for group assignment
  - Needs https://github.com/cvat-ai/cvat/issues/40 to be addressed

### Checklist
- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- ~~[ ] I have added tests to cover my changes~~
- ~~[ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
- ~~[ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning), [cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))~~
### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by nmanovic** [comment]
@RyanHir , thanks for the contribution. Please let us know when the PR is ready. We will be happy to review and merge it!

**Comment by RyanHir** [comment]
Hello @nmanovic,

I believe I have everything ready for review. However, I am unable to test if groups work. I opened an issue at #40 discussing this.

**Comment by RyanHir** [comment]
@vaskokj you said in https://github.com/openvinotoolkit/cvat/issues/311 that you got FreeIPA auth working. Do you mind looking over this example, as you redacted a fair bit of information in your config?

**Comment by vaskokj** [comment]
@RyanHir I updated the other ticket with new information. Hopefully that gets you headed in the right direction.

**Comment by RyanHir** [comment]
> Hello @nmanovic,
> 
> I believe I have everything ready for review. However, I am unable to test if groups work. I opened an issue at #40 discussing this.

Everything appears to be in order now

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Tests for import/export annotation, dataset, backup from/to cloud storage
 - Added Python SDK package (`cvat-sdk`)
 - Previews for jobs
+- Documentation for LDAP authentication
```
Comment (nmanovic) [inline CHANGELOG.md:21]:
```suggestion
- Documentation for LDAP authentication (<https://github.com/cvat-ai/cvat/pull/39>)
```

File: site/content/en/docs/administration/advanced/ldap.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+}
+```
+
+### Resources
+- [Microsoft - LDAP Distinguished Names](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/ldap/distinguished-names)
+  - Elements that make up a distinguished name. Used with user/group searches.
+- [Django LDAP Reference Manual](https://django-auth-ldap.readthedocs.io/en/latest/reference.html)
+  - Other options that can be used for LDAP authentication in Django.
+- [Django LDAP guide using Active Directory (Unofficial)](https://techexpert.tips/django/django-ldap-authentication-active-directory)
+  - This is not specific to CVAT but can provide insight about firewall rules.
```
Comment (nmanovic) [inline site/content/en/docs/administration/advanced/ldap.md:210]:
```suggestion
  - This is not specific to CVAT but can provide insight about firewall rules.
  
```

**Review by nmanovic — APPROVED** [review]
@RyanHir , great job! Awesome documentation.

**Pull request merged**